From 06f6db28648cdeff02cc07f9a7f1410b228450a4 Mon Sep 17 00:00:00 2001
From: monsta <monsta@inbox.ru>
Date: Wed, 14 Sep 2016 18:20:25 +0300
Subject: [PATCH] check for libindicator 12.10.2 for new style indicators
 support

fixes build on Gentoo which has 12.10.1 - turned out this version
had no indicator-ng yet.

note that (as of now) 12.10.2 isn't officially released upstream,
and Ubuntu has post-12.10.1 repo snapshot packaged and labeled as
12.10.2. however, that snapshot already has 12.10.2 version in .pc
file, so the build works with it.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 5e9c505..9b4dafc 100644
--- a/configure.ac
+++ b/configure.ac
@@ -46,7 +46,7 @@ case "$with_gtk" in
 esac
 
 INDICATOR_REQUIRED_VERSION=0.3.90
-INDICATOR_NG_VERSION=12.10
+INDICATOR_NG_VERSION=12.10.2
 INDICATOR_PKG=indicator$INDICATOR_API_VERSION-0.4
 
 PKG_CHECK_MODULES(APPLET, gtk+-$GTK_API_VERSION >= $GTK_REQUIRED_VERSION
